public/index.php

```diff
@@ -1,9 +1,19 @@
 <?php
 
+use App\Database;
 use App\Kernel;
 
 require_once __DIR__ . '/../vendor/autoload.php';
 
-$Kernel = new Kernel();
+$database = new Database('witryna1db');
```

Comment:
zbędne, nie przekazuj do konstruktora tylko wywołaj to w metodzie initConnection w pliku Kernel.php i przypisz do właściwości $this->database = new Database...